creator-node/compose/env/base.env

```diff
@@ -35,15 +35,13 @@ dataProviderUrl=http://audius_ganache_cli:8545
 spOwnerWalletIndex=
 
 
-# Sync / SnapbackSM configs
-disableSnapback=true
+# State Machine configs
 stateMonitoringQueueRateLimitInterval=60000
 stateMonitoringQueueRateLimitJobsPerInterval=3
-snapbackModuloBase=3
 minimumDailySyncCount=5
 minimumRollingSyncCount=10
 minimumSuccessfulSyncCountPercentage=50
-snapbackHighestReconfigMode=PRIMARY_AND_OR_SECONDARIES
+stateMachineHighestReconfigMode=PRIMARY_AND_OR_SECONDARIES
```

Comment:
left one comment about this on https://github.com/AudiusProject/audius-docker-compose/pull/56